lib/hsa/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
+include(FindPkgConfig)
+
 ####################
 # HCC runtime (HSA implementation)
 ####################
 if (HAS_ROCM EQUAL 1)
 add_mcwamp_library_hsa(mcwamp_hsa mcwamp_hsa.cpp unpinned_copy_engine.cpp)
-target_link_libraries(mcwamp_hsa  unwind)
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
+# TODO: we should probably add required CFLAGS and LDFLAGS from pkgconfig
+target_link_libraries(mcwamp_hsa ${LIBUNWIND_LIBRARIES})
```

Comment:
what's the advantage of imported target if we're not using any of the **IMPORTED_** properties?
_hcc-config.cmake.in_ is only preprocessed and not included anywhere so cmake commands won't work.
If I try to include the resulting _hcc-config.cmake_ just after the call to _configure_package_config_file_ it fails even without any changes because it uses _set_and_check_ on install directories which don't exists at configure time (it also tries to check for _/home/include_ which looks wrong)